### ISSUE TITLE
"Update user status" request payload is incorrect

### ISSUE THREAD

**Issue by maruTA-bis5** [mattermost/mattermost-api-reference#416]
"Update user status" document's request payload is incorrect.
https://api.mattermost.com/#tag/status%2Fpaths%2F~1users~1%7Buser_id%7D~1status%2Fput

I got this error response when request same as api reference.
- request
```
1 17, 2019 7:34:22 午前 org.glassfish.jersey.logging.LoggingInterceptor log
警告: 25 * Sending client request on thread main
25 > PUT http://192.168.10.32:8065/api/v4/users/e6g6n7a7jtnmbg1rqinek8hkxh/status
25 > Accept: application/json
25 > Authorization: Bearer 99ep9a8ynbgwtgx6cxsfo8dn7y
25 > Content-Type: application/json
{"status":"online"}
```
- response
```
1 17, 2019 7:34:22 午前 org.glassfish.jersey.logging.LoggingInterceptor log
警告: 25 * Client response received on thread main
25 < 400
25 < Content-Length: 184
25 < Content-Type: application/json
25 < Date: Wed, 16 Jan 2019 22:34:22 GMT
25 < X-Request-Id: i5w59hkgrtr19p8myt543uqwoe
25 < X-Version-Id: 5.6.0.5.6.2.79239b179a0fefa54beb1a4b2aa5493e.false
{"id":"api.context.invalid_body_param.app_error","message":"Invalid or missing user_id in request body","detailed_error":"","request_id":"i5w59hkgrtr19p8myt543uqwoe","status_code":400}
```

**Comment by amyblais** [comment]
Thanks @maruTA-bis5, I opened a bug ticket for this here: https://mattermost.atlassian.net/browse/MM-13810.

**Comment by amyblais** [comment]
Closing this as the above ticket has been resolved for upcoming v5.9 release,
